metro.config.js

```diff
@@ -1,10 +1,22 @@
-module.exports = {
-    transformer: {
-        getTransformOptions: async () => ({
-            transform: {
-                experimentalImportSupport: false,
-                inlineRequires: false,
+const { getDefaultConfig } = require('metro-config');
+
+module.exports = (async () => {
+    const {
+        resolver: { sourceExts, assetExts },
+    } = await getDefaultConfig();
+        return {
+            transformer: {
+                babelTransformerPath: require.resolve('react-native-svg-transformer'),
+                getTransformOptions: async () => ({
```

Comment:
Unnecessary async?

Reply:
This is the default by react-native when you init a project. Maybe they have it there to show that it can be an async func. But yeah, we don't need it